yaml/virtual-copy/virtual-copy-base-vol-tc2.yml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: ci-primera3par-csp-secret
+  name: hpe3parprimera-secret-tc2
```

Comment:
In virtual copy yaml folder, the secret name has been changed to the old name. I had modified all the secret name to match the name in Conftest.py file. Not sure why this is changed back

Reply:
Addressed the comment and made appropriate changes. Please review.